src/js/app.js

```diff
@@ -0,0 +1,67 @@
+$(document).ready(function() {
+
+
+
+
+$('#searchBtn').on('click', function(event) {
+  event.preventDefault();
+
+
+  $.get('https://www.reddit.com/search.json', {
+
+    q: searchInput,
+  }).done(function(data) {
+    intoArray(data);
+    // console.log(data);
+  }).fail(function(data) {
+    console.log('failed!');
+  });
```

Comment:
Nice job adding some error handling. This is where you may also want to display a message to the user. But a console log works as a first step